Neos.ContentRepository.TestSuite/Classes/Behavior/Features/Bootstrap/CRTestSuiteTrait.php

```diff
@@ -154,7 +156,20 @@ public function workspaceHasStatus(string $rawWorkspaceName, string $status): vo
      */
     public function iExpectTheGraphProjectionToConsistOfExactlyNodes(int $expectedNumberOfNodes): void
     {
-        $actualNumberOfNodes = $this->currentContentRepository->getContentGraph($this->currentWorkspaceName)->countNodes();
+        // HACK to access
+        $contentRepositoryReadModelAccess = new class implements ContentRepositoryServiceFactoryInterface {
+            public ContentRepositoryReadModel|null $instance;
+            public function build(ContentRepositoryServiceFactoryDependencies $serviceFactoryDependencies): ContentRepositoryServiceInterface
+            {
+                $this->instance = $serviceFactoryDependencies->projectionsAndCatchUpHooks->readModelProjection->getState();
+                return new class implements ContentRepositoryServiceInterface
+                {
+                };
+            }
+        };
```

Comment:
We could declare some custom Test CR Service that just exposes all `ContentRepositoryServiceFactoryDependencies` to make these less of a hack – but that can be done in a follow-up of course